packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -74,6 +74,40 @@ new codepipeline.JenkinsBuildAction(this, 'Jenkins_Build', {
 });
 ```
 
+### Cross-region CodePipelines
+
+You can also use the cross-region feature to deploy resources
+(currently, only CloudFormation Stacks are supported)
+into a different region than your Pipeline is in.
+
+It works like this:
+
+```ts
+const pipeline = new codepipeline.Pipeline(this, 'MyFirstPipeline', {
+  // ...
+  crossRegionReplicationBuckets: {
+    'us-west-1': 'my-us-west-1-replication-bucket',
+  },
+});
+
+// later in the code...
+new cloudformation.PipelineCreateUpdateStackAction(this, 'CFN_US_West_1', {
+  // ...
+  region: 'us-west-1',
+});
+```
+
+This way, the `CFN_US_West_1` Action will operate in the `us-west-1` region,
+regardless of which region your Pipeline is in.
+
+If you don't provide a Bucket name for a region (other than the Pipeline's region)
+that you're using for an Action with the `crossRegionReplicationBuckets` property,
+there will be a new Stack, named `aws-cdk-codepipeline-cross-region-scaffolding-<region>`,
+created for you, containing a replication Bucket.
```

Comment:
"created" => "defined" (that's the term we use, because you don't really create things with the CDK)

Reply:
Changed.